### PR TITLE
[ADD] events: event booths doc (17)

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3928740&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

This PR is focused on informing the reader about everything they'll need to know when it comes to event booth creation and management. There was no such documentation about this specific subject yet.

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hey @brse-odoo -- this one is ready for a Peer Review from you whenever you get a chance. Thanks! 👍

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:46]:
```suggestion
- :guilabel:`Create Sponsor`: if the box in this column is ticked, when a booth with this
```

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:53]:
```suggestion
resulting drop-down menu, tick the checkbox beside :guilabel:`Sponsor Level` and/or :guilabel:`Sponsor
```

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:84]:
```suggestion
that checkbox is ticked, when a booth belonging to this booth category is booked,
```

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:87]:
```suggestion
When the :guilabel:`Sponsorship` checkbox is ticked, two additional fields appear beneath it:
```

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:94]: Is there an fa-icon that this guilabel emoji can be replaced with? So that we're consistently using either odoo icons or fa-icons throughout the doc

Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:94]:
```suggestion
   the :guilabel:`➡️ (right arrow)` that appears at the end of the line. Doing so opens a separate page,
```

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:108]:
```suggestion
of :guilabel:`Ribbon Style` should be applied, if any. The :guilabel:`Ribbon Style` options available
```

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:112]:
```suggestion
If one is selected, that :guilabel:`Ribbon Style` appears with the sponsor's name on the event
```

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:122]:
```suggestion
booth category that would be important for any potential booth-buyer to know about (e.g., the square
```
Comma after e.g. when multiple examples are presented

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:140]: Question about the graph view - is it represented by this Odoo UI icon? : 

oi-view-cohort

I think it might be, if it is I'd say to use that one

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:145]:
```suggestion
the event. The booths present in the :guilabel:`Unavailable` stage have already been purchased, and are no
```

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:178]:
```suggestion
appear: :guilabel:`Product` and :guilabel:`Price`. Both fields represent their respective
```

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:181]:
```suggestion
When a person purchases a booth rental through the event website, the subsequent renter-related fields
```
Ignore if incorrect

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:185]:
```suggestion
However, if the rental of a booth is conducted in any other way (e.g., in person, via sales order,
```

Comment (brse-odoo) [reply content/applications/marketing/events/event_booths.rst:185]: No hyphen for in person if not before a noun

File: content/applications/marketing/events/event_booths.rst
Comment (brse-odoo) [inline content/applications/marketing/events/event_booths.rst:220]:
```suggestion
   If the selected booth has the *Create Sponsor* checkbox ticked, this page reads as *Sponsor
```

**Review by brse-odoo — APPROVED** [review]
@ksc-odoo I've finished my review of this PR, great job! I'm approving with suggestions for you to accept/reject as you see fit. Let me know if you have any questions, thanks!

File: content/applications/marketing/events/event_booths.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/event_booths.rst:140]: Actually, oi-view-cohort is for the cohort view. the FA icon I have here represents the appropriate icon, so I'm going to stick with that, unless SALI instructs me to change it.

**Comment by ksc-odoo** [comment]
Thanks for the helpful feedback, @brse-odoo -- made all your suggested edits. This one is ready for a Final Review, @StraubCreative 👍

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:15]:
```suggestion
To do that, navigate to :menuselection:`Events app --> Configuration --> Settings`, and tick
:guilabel:`Booth Management` checkbox. Then, click :guilabel:`Save`.
```

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:23]: optionally, can link to this doc (:
https://www.odoo.com/documentation/17.0/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.html#set-product-type

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:37]: Totally optional: slight rewording for brevity

With the Booth Management setting activated in the Events app, the Booth Categories option appears in the :guilabel:Configuration menu.

To access it, go to :menuselection:Events app --> Configuration --> Booth Categories, revealing a list of all created booth categories.

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:47]:
```suggestion
- :guilabel:`Create Sponsor`:  If checked, booking this booth category creates a sponsor for the user.
```

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:51]:
```suggestion
When the :icon:`oi-settings-adjust` :guilabel:`(settings)` icon, located to the far-right of
```

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:54]: What do the Sponsor Level and Type fields do? is there another doc that elaborates on these?

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:78]: optional, for brevity
```suggestion
In the :guilabel:`Booth Details` section, users **must** assign a :guilabel:`Product`, with *Event Booth* set as the *Product Type* on the product form.
```

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:85]: While I do see a necessity to explain what this field is on the form, we've defined it already in line 47. Is it necessary to repeat that? o:

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:90]: what do the different sponsor levels do for an event booth? Why is it necessary to make a distinction between tiers? 

While I understand that Odoo documentation is function-focused, some context here may help a reader who doesn't know Event Booths as well as you do. (:

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:116]: which drop-down menu are we referring to right now? I'm in the Create Sponsor Level one, and I don't see "Create".
image

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:201]: If the "Go to Website" smart button is not available, and users see the "Unpublished" smart button, what would they do? :eyes:

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:204]: where is the "Get a Booth" event subheader menu? o;
image

Also, are only "Unavailable" booths able to be booked?

**Review by Felicious — CHANGES_REQUESTED** [review]
Hi @ksc-odoo! Wonderful doc so far! It's very detailed and easy to understand (:

I have minor suggestions to reword for brevity, that you may consider. After that, I ran into an issue finding the "Get a Booth" event subheader menu. Maybe I missed something during the setup?

Happy to prioritize this doc first thing tomorrow morning and figure out what went wrong with my setup, with your help :smile: 

Have a good rest of your morning!

File: content/applications/marketing/events/event_booths.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/event_booths.rst:54]: In this case, it simply reveals those specific columns. So, it would merely display the Sponsor Level and Sponsor Type on the Booth Categories page. While those are somewhat self-explanatory, I agree, that a sponsor-related doc would be nice. I'll add it to my to-do list, and knock it out at some point in the near future.

File: content/applications/marketing/events/event_booths.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/event_booths.rst:78]: I know that my version is slightly longer, but I think it's more just a *tiny bit* easier for the reader to comprehend. So, if it's okay with you, I think I'm going to stick with the current wording. However, if you feel like I should rework this portion, just let me know, and I'll make the necessary adjustments.

File: content/applications/marketing/events/event_booths.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/event_booths.rst:85]: I completely understand (and appreciate) this suggestion. I agree, this has been covered already, just a few lines above. However, in the random event of a reader skimming through the doc to locate a 'Create booth category' section, they may not have looked at that portion, and would still find all the necessary information here. So, again, if it's okay with you, I think I'll leave this in for now, just so the potential "jump around reader" can still have all the information they need to succeed 👍

File: content/applications/marketing/events/event_booths.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/event_booths.rst:90]: I added an admonition briefly explaining these concepts, per your request. 👍

File: content/applications/marketing/events/event_booths.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/event_booths.rst:116]: This admonition was in a poor position in the doc, which caused this totally understandable confusion. My apologies for that. I have since moved it to a far more appropriate position in the doc, and it makes a lot more sense now. Thank you for bringing this to my attention 👍

**Comment by StraubCreative** [comment]
@ksc-odoo to merge this in please address @Felicious feedback and pass to @samueljlieber after, thanks!

File: content/applications/marketing/events/event_booths.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/event_booths.rst:208]: In my experience, the 'Go to Website' smart button is routinely available, however, as you've mentioned, it can oftentimes be Red/Unpublished. Fortunately, that doesn't affect the ability to click that smart button to reveal the front-end of the event website. So, taking that into consideration, I think I'll be able to keep this as-is, mainly because I believe the concept of 'publishing' an event for all to see is covered in another doc. But, once again, I sincerely appreciate the comment and your amazing attention to detail. Thanks! 👍

File: content/applications/marketing/events/event_booths.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/event_booths.rst:211]: Woo hoo! I figured it out. And, GREAT CATCH on this! I've added a detailed admonition, with specific steps, for 2 different methods users can employ to ensure they see the 'Get A Booth' subheader menu option. It should definitely solve any issues they encounter. Thanks again for bringing this to my attention. 🙏

**Comment by ksc-odoo** [comment]
Hey @Felicious -- thank you SO MUCH for the detailed feedback. I really appreciate it. I believe I was able to complete all your suggested edits successfully, so if you could give this another quick look at your earliest convenience, that would be great. Once you approve, I'll send this over to SALI for Tech Review. Thanks again 👍 🙏

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:238]: This content looks great, and I'm glad you were able to figure out what makes the event subheader appear :smile: 

Given that we're using a pre-configured runbot, the Event subheader menu appears by default. However, for general users creating a new, blank event, this subheader menu won't be visible. To align with their experience, I suggest moving this information out of the note admonition and placing it below your instruction to click the "Go to Website" button. After enabling the event subheader, we can guide them to click the "Get a Booth" button.

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:205]: I'm going to nitpick for a bit:

While it's FANTASTIC that you've been deepening your knowledge in developer concepts, terms like "front-end" and "backend" are typically used in programming to refer to server-side code (backend) and webpage-side code (front-end).
image
[For more info, here's an article](https://aws.amazon.com/compare/the-difference-between-frontend-and-backend/)

Odoo simplifies website development by allowing users to modify webpages through the Odoo app, removing the need for code. Because of this abstraction, it's no longer suitable to refer to what we do in Odoo as front and back-end.

For our non-developer audience, it might be more technically correct to use:
backend --> event form
front-end --> event website preview

If you agree with my feedback, I'd use CTRL+F to find all instances of "backend" and "frontend" and make the adjustments :blush:

File: content/applications/marketing/events/event_booths.rst
Comment (Felicious) [inline content/applications/marketing/events/event_booths.rst:240]: I wasn't able to Book a booth without a location. 
```suggestion
From here, the visitor can select their desired booth option, then :guilabel:`Location`
```

**Review by Felicious — APPROVED** [review]
Hi @ksc-odoo, great job with this new event booths doc :blush: 

I reviewed the last 20 lines of your doc that I couldn't get to last time, and everything works as expected! 

After you review my comments, feel free to pass this onto the next stage!

**Comment by ksc-odoo** [comment]
Thanks for the always helpful, super in-depth feedback, @Felicious ! I really appreciate it. I made all those necessary adjustments, and believe this one is now ready for Tech Review, @samueljlieber (whenever you get a chance). Thanks! 👍

File: content/applications/marketing/events/event_booths.rst
Comment (StraubCreative) [inline content/applications/marketing/events/event_booths.rst:46]: `content/applications/marketing/events/event_booths.rst:46: Line too long (101/100) (line-too-long)`

File: content/applications/marketing/events/event_booths.rst
Comment (StraubCreative) [inline content/applications/marketing/events/event_booths.rst:221]: These numbered list items are not formatted correctly.

**Review by StraubCreative — APPROVED** [review]
Hi @ksc-odoo 
Just a couple items for you and can merge after, thanks!
.....
@robodoo delegate=ksc-odoo

**Comment by ksc-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
